code/__DEFINES/layers.dm

```diff
@@ -111,21 +111,21 @@
 #define HIGH_TURF_LAYER 2.03
 #define TURF_PLATING_DECAL_LAYER 2.031
 #define TURF_DECAL_LAYER 2.039 //Makes turf decals appear in DM how they will look inworld.
-#define DISPOSAL_PIPE_LAYER 2.04
-#define WIRE_LAYER 2.041
-#define ABOVE_OPEN_TURF_LAYER 2.042
+#define LATTICE_LAYER 2.04
+#define DISPOSAL_PIPE_LAYER 2.042
+#define WIRE_LAYER 2.044
+#define GLASS_FLOOR_LAYER 2.046
+#define TRAM_RAIL_LAYER 2.047
+#define TRAM_FLOOR_LAYER 2.048
+#define ABOVE_OPEN_TURF_LAYER 2.049
 
```

Comment:
wires aren't on the floor plane, why does this work?

Reply:
They're on the floor plane.

Reply:
When did that happen

Reply:
https://github.com/tgstation/tgstation/pull/72389